tools/normalize_mri_protocol_range_data.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * This script is the part of the second step for normalizing the mri_protocol table.
+ * It populates the new min & max columns for each field that can contain ranges and
+ * returns SQL statements to remove the old %_range columns.
```

Comment:
and returns SQL DROP statements to remove the old %_range columns if the option "tosql" is set

(or whatever the name of that option will be)

Reply:
Might also specify that they first need to run the patch before running the script.